skbeam/core/tests/test_roi.py

```diff
@@ -318,22 +318,30 @@ def test_circular_average():
                                                max_x=10, nx=None)
     assert_array_almost_equal(bin_cen1, [0.5, 1.5, 2.5, 3.5, 4.5, 5.5, 6.5,
                                          7.5, 8.5])
+
     mask = np.ones_like(image)
     mask[4:6, 2:3] = 0
 
+    simg = np.zeros_like(image)
+
     bin_cen_masked, ring_avg_masked = roi.circular_average(image,
                                                            calib_center,
                                                            min_x=0,
                                                            max_x=10,
                                                            nx=6,
-                                                           mask=mask)
+                                                           mask=mask,
+                                                           SIMG=simg)
```

Comment:
sorry, forgot to update the tests. I did but I still need to check this works with anisotropic pixels later, tonight.. thanks!